diff --git a/messages.c b/messages.c
index 469e381..770d9ad 100644
--- a/messages.c
+++ b/messages.c
@@ -477,10 +477,12 @@ int cmd_desktop(char **args, int num)
 				else
 					return MSG_FAILURE;
 			}
-			if (streq("floating", key))
+			if (streq("floating", key)) {
 				trg.desktop->floating = (a == ALTER_SET ? b : !trg.desktop->floating);
-			else
+				put_status(SBSC_MASK_REPORT);
+			} else {
 				return MSG_FAILURE;
+			}
 		} else {
 			return MSG_SYNTAX;
 		}
diff --git a/subscribe.c b/subscribe.c
index 9931a8e..969724c 100644
--- a/subscribe.c
+++ b/subscribe.c
@@ -88,7 +88,7 @@ int print_report(FILE *stream)
 			fprintf(stream, "%c%s:", c, d->name);
 		}
 		if (m->desk != NULL)
-			fprintf(stream, "L%c%s", (m->desk->layout == LAYOUT_TILED ? 'T' : 'M'), (m != mon_tail ? ":" : ""));
+			fprintf(stream, "L%c:4%c%s", (m->desk->layout == LAYOUT_TILED ? 'T' : 'M'), (m->desk->floating ? 'y' : 'n'), (m != mon_tail ? ":" : ""));
 	}
 	fprintf(stream, "%s", "\n");
 	return fflush(stream);
