diff --git a/config.def.h b/config.def.h
index ac32bd7..5fac653 100644
--- a/config.def.h
+++ b/config.def.h
@@ -56,6 +56,9 @@ static const bool ALPHA_LAYER = false;
 /* thumbnail sizes in pixels (width == height): */
 static const int thumb_sizes[] = { 32, 64, 96, 128, 160 };
 
+/* default zoom level for thumbnail mode */
+static const int THUMB_ZOOM_LEVEL = 0;
+
 #endif
 #ifdef _MAPPINGS_CONFIG
 
diff --git a/thumbs.c b/thumbs.c
index 209c94b..096dd73 100644
--- a/thumbs.c
+++ b/thumbs.c
@@ -175,7 +175,7 @@ void tns_init(tns_t *tns, fileinfo_t *files, const int *cnt, int *sel,
 	tns->win = win;
 	tns->dirty = false;
 
-	tns->zl = 0;
+	tns->zl = THUMB_ZOOM_LEVEL;
 	tns_zoom(tns, 1);
 
 	if ((homedir = getenv("XDG_CACHE_HOME")) == NULL || homedir[0] == '\0') {
